src/api.js

```diff
@@ -844,6 +912,48 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
         return stmt;
     };
 
+    /** Iterate over multiple SQL statements in a SQL string
+     @param {string} sql a string of SQL
+     @return {StatementIterator} the resulting statement iterator
+     */
+    Database.prototype["iterateStatements"] = function iterateStatements(sql) {
```

Comment:
Ideally this would return an `Iterable` and not an Iterator so that we can use `for .. of`. But we need to be compatible with old runtimes, so we don't want to crash when Symbol.iterator is not available and want to have a way to get the iterator without that.

Reply:
I'm not well versed enough in this area to be confident I know how to do this; certainly not enough to know how to test it on different runtimes :(

Reply:
This should be a simple 


```js
if (typeof Symbol == "function" && typeof Symbol.iterator == "symbol") {
  StatementIterator.prototype[Symbol.iterator] = function iterator() {
    return this;
  }
}
```